packages/ws-server/src/webSocketServer.ts

```diff
@@ -120,6 +120,8 @@ app.ws.use((ctx) => {
     ctx.websocket.on('close', function () {
         relay.subs()?.unsubscribe(ctx.websocket);
         console.log('stopping client interval');
+        connectedClients--;
```

Comment:
Not sure if this change originates in this PR or in 1011 but I posed a questions around it here
https://github.com/hashgraph/hedera-json-rpc-relay/pull/1011/files#r1150967792

Reply:
I copied from here, since my tests were failing due to the connections of the previous tests were not being closed...

Reply:
leaving original question from Nana here:

Q: didn't we explore this earlier and find out that connectedClients = ctx.app.server._connections; captures increments and decrements?

If this is not the case we should add a comment saying decrements aren't handled.

Also should the decrement if applicable occur after the termination to avoid inconsistencies if termination doesn't occur.

Reply:
We don't have access to the ctx.app.server._connections in the verifyClient call back, hence the need to track it in a class variable.  I added tests around the number of connections after opening and closing multiple connections.

Reply:
Okay please add a comment above the decrement to be clear why we decrement but never increment.

Reply:
Done.